lib/data/odata-filter.js

```diff
@@ -87,5 +87,27 @@ const odataFilter = (expr, odataToColumnMap) => {
   return op(ast);
 };
 
-module.exports = { odataFilter };
+const odataOrderBy = (expr, odataToColumnMap, stableOrderColumn = null) => {
+  if (expr == null) return sql`true`;
+
+  let initialOrder = null;
+  const clauses = expr.split(',').map((exp) => {
+    const [col, order] = exp.trim().split(/\s+/);
+    if (!odataToColumnMap.has(col))
+      throw Problem.internal.unsupportedODataField({ at: 0, text: col });
+
+    // Save the order of the initial property to use for the stable sort column order
+    if (initialOrder == null)
+      initialOrder = order;
+
+    return sql`${sql.identifier(odataToColumnMap.get(col).split('.'))} ${((order?.toLowerCase() === 'asc') ? sql`ASC` : sql`DESC`)}`;
```

Comment:
odata spec (https://learn.microsoft.com/en-us/azure/search/search-query-odata-orderby) mentions " If there are null values in the field, null values appear first if the sort is asc and last if the sort is desc."

Add `NULLS FIRST` to end of each sort clause.